libgeopmd/libgeopmd.spec.in

```diff
@@ -27,7 +27,6 @@ BuildRequires:	gcc-c++
 BuildRequires:	glibc-devel
 BuildRequires:	libcap-devel
 BuildRequires:	libtool
-BuildRequires:	liburing-devel
```

Comment:
Doesn't configure.ac expect that liburing-devel is present by default (without --disable-io-uring)?

Reply:
Yes, but i had listed it twice in the spec file, this change deletes the unconditional requirement